R/scale-gradient.r

```diff
@@ -54,6 +62,19 @@
 #'   scale_colour_gradient(low = "white", high = "black")
 #' # Avoid red-green colour contrasts because ~10% of men have difficulty
 #' # seeing them
+#'
+#' # Pass na.value = NA with the fill aesthetic to result in no fill for missing values
```

Comment:
Suggested change which highlights the fact that the original axis range is kept:

```
# Use `na.value = NA` to hide missing values 
# but keep the original the axis range
```